docs/data/routes/docs/client-frameworks/nextjs/nextjs-static-export/en.md

```diff
@@ -0,0 +1,42 @@
+---
+name: nextjs-static-export
+routeTemplate: ./data/component-templates/article.yml
+title: Next.js static export
+---
+
+# Next.js Static HTML Export
+
+Using JSS Next.js sample app you can run `jss export` that allows you to export your app to static HTML, which can be run standalone without the need of a Node.js server. Read more about Next.js [Static HTML Export](https://nextjs.org/docs/advanced-features/static-html-export).
+
+> Currently here are limitations for usage of Static Export: 
```

Comment:
1. Might be good to callout whether each limitation is Next.js' or Sitecore's.
2. Add Next.js Image Optimization (next/image) limitation
3. Should we remove the API Routes one?

Reply:
@ambrauer As Nick mentioned, we will not remove api routes, as I understood
And we are not using Image Optimization even in SSG, should we mention it?

Reply:
@sc-illiakovalenko I was referring to the limitation bullet "API Routes are not supported because they can't be prerendered to HTML.". Should that be mentioned still?

True, maybe it's best that we don't mention Image Optimization since we no longer include it. I think we should be clear then that this list is only the limitations of our sample app. e.g. "Currently, these are the limitations of the Next.js sample app for usage of Static HTML Export:"

Reply:
@ambrauer Renamed `restrictions` - first line